static/locales/fr/translation.json

```diff
@@ -131,15 +131,15 @@
 		"confirm": "Valider",
 		"cancel": "Annuler",
 		"close": "Fermer",
-		"exit": "Quittez les Paramètres pour Appliquer les Changements",
-		"public_lobby": "Lobbies Publiques",
+		"exit": "Quittez les pParamètres pour appliquer les changements",
+		"public_lobby": "Parties publiques",
```

Comment:
Again, it's a lobby (the place that people can go) not a party/group (the people in a lobby)